akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -384,10 +387,13 @@ private[remote] class Association(
       } catch {
         case ShuttingDown ⇒ // silence it
       }
-    } else if (log.isDebugEnabled)
-      log.debug(
-        "Dropping message [{}] from [{}] to [{}] due to quarantined system [{}]",
-        Logging.messageClassName(message), sender.getOrElse(deadletters), recipient.getOrElse(recipient), remoteAddress)
+    } else {
+      droppedMessageLogging.filter { discardInfo ⇒
+        log.warning(
+          s"Dropping message [{}] from [{}] to [{}] due to quarantined system [{}]$discardInfo",
```

Comment:
We might be losing the exact point of time here when the log message changes from `due to {}` to `due to quarantined system` because of the discarded messages. Maybe it's not important enough but otherwise it could be confusing.